src/runtime_src/core/common/system.cpp

```diff
@@ -2,12 +2,12 @@
 // Copyright (C) 2019 Xilinx, Inc
 // Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
 #define XRT_CORE_COMMON_SOURCE
+// Local - Include Files
 #include "system.h"
 #include "device.h"
 #include "module_loader.h"
-#include "gen/version.h"
```

Comment:
@AShivangi , why did you remove gen/version.h?  get_xrt_build_info() uses it.  At least on my system it breaks the build.  I really bumped it passed the pipeline, and mystified why it is failing to build on my ubuntu20.04 system.  I am adding it back in.